mappings/net/minecraft/client/particle/Particle.mapping

```diff
@@ -75,7 +75,7 @@ CLASS bnl net/minecraft/client/particle/Particle
 		ARG 2 z
 	METHOD b setSpriteIndex (I)V
 		ARG 0 value
-	METHOD c hasTransparency ()Z
+	METHOD c isTransparent ()Z
```

Comment:
Does this mean the particle has some fully transparent pixels or partial transparency?